packages/cells/src/widget.ts

```diff
@@ -716,12 +729,19 @@ export class CodeCell extends Cell<ICodeCellModel> {
   constructor(options: CodeCell.IOptions) {
     super(options);
     this.addClass(CODE_CELL_CLASS);
+    const trans = this.translator.load('jupyterlab');
 
     // Only save options not handled by parent constructor.
     const rendermime = (this._rendermime = options.rendermime);
     const contentFactory = this.contentFactory;
     const model = this.model;
 
+    // Note that modifying the below label warrants one to also modify
+    // the same in this._outputLengthHandler. Ideally, this label must
+    // have been a constant and used in both places but it is not done
+    // so because of limitations in the translation manager.
+    let ariaLabel = 'Code Cell Content with Output';
```

Comment:
it feel unnatural as mentioned by @krassowski. But you need to wrap the string with the translation method for it to be extracted from the code. It does not find the strings if you request the translation of a variable.

```suggestion
    let ariaLabel = trans.__('Code Cell Content with Output');
```

Reply:
Thanks for the super-fast review @fcollonval !

Oops. From the previous comment, `define the constants somewhere where you have access to trans`, I assumed this would work if the variables are reachable by `trans`. I've added all your suggestions with the recent commit.